apps/wallet/src/ui/app/shared/text/index.tsx

```diff
@@ -21,6 +21,7 @@ const textStyles = cva([], {
             caption: 'uppercase text-caption',
             captionSmall: 'uppercase text-captionSmall ',
             heading4: 'text-heading4',
+            heading6: 'text-heading6',
```

Comment:
Why not use the heading component? Feels wrong to have heading styles here. I'd remove the `heading4` that got added here as well.